modules/govuk_ci/manifests/agent/elasticsearch.pp

```diff
@@ -0,0 +1,31 @@
+# == Class: govuk_ci::agent::elasticsearch
+#
+# Installs and configures elasticsearch
+#
+# === Parameters
+#
+# [*conf_maxmemory*]
+#   Sets the config option "maxmemory" for redis
+#
+class govuk_ci::agent::elasticsearch(
+  $heap_size = '64m',
+  $elasticsearch_version = '1.7.5',
+) {
+  include ::govuk_java::oracle7::jre
```

Comment:
I think this should run under `govuk_java::openjdk7::jre` because that's what the `govuk::node::s_api_elasticsearch` uses.